golang/pkg/cli/runner.go

```diff
@@ -88,6 +90,16 @@ func ProcessCommand(ctx context.Context, initialState *State, args *ArgsFlags) {
 	case DownCommand:
 		StopContainers(ctx, args)
 		log.Info().Msg("Stack is stopped. Hope you had fun! 🎬")
+	case VersionCommand:
+		cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+		if err != nil {
+			log.Fatal().Err(err).Msg("Could not connect to docker socket.")
+		}
+		out, err := container.VersionCheck(ctx, cli)
+		if err != nil {
+			log.Fatal().Err(err).Msg("Version error")
+		}
+		out.Msg("")
```

Comment:
Why need an empty message?

Reply:
Something to terminate / print the zerolog event must be called for it to be visible.

Reply:
`Send()` ?

Reply:
We can agree on using not using Msg("") or Msgf("") but then it must be a lint rule too.